airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -560,7 +562,7 @@ def _submit_job(
         configuration = {"query": {"query": sql, "useLegacySql": self.use_legacy_sql}}
         return hook.insert_job(
             configuration=configuration,
-            project_id=hook.project_id,
+            project_id=self.bq_project_id if self.bq_project_id else hook.project_id,
```

Comment:
```suggestion
            project_id=self.bq_project_id or hook.project_id,
```

Reply:
Will work on this fix and send out the updated one for review

Reply:
fixed